distributed/dashboard/components/scheduler.py

```diff
@@ -559,6 +559,140 @@ def update(self):
         update(self.source, d)
 
 
+class Hardware(DashboardComponent):
+    """Occupancy (in time) per worker"""
+
+    def __init__(self, scheduler, **kwargs):
+        with log_errors():
+            self.scheduler = scheduler
+            # Disk
+            self.disk_source = ColumnDataSource(
+                {
+                    "size": [],
+                    "bandwidth": [],
+                }
+            )
+
+            self.disk_figure = figure(
+                title="Disk Bandwidth -- Computing ...",
+                tools="",
+                toolbar_location="above",
+                x_range=["1 kiB", "100 kiB", "1 MiB", "10 MiB", "100 MiB"],
```

Comment:
This breaks as soon as this list is not a perfect copy-paste of the one in worker.py.
If possible, you should avoid pre-defininig the x_range (like nbytes does).
Otherwise, please define a constant in worker.py and import it in this module.

Reply:
I'm handling this dynamically now using Bokeh FactorRanges.